content/en/docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -52,7 +52,7 @@ This may be caused by a number of problems. The most common are:
   Check the system log file (e.g. `/var/log/message`) or examine the output from `journalctl -u kubelet`. If you see something like the following:
 
   ```shell
-  error: failed to run Kubelet: failed to create kubelet: 
+  error: failed to run Kubelet: failed to create kubelet:
   misconfiguration: kubelet cgroup driver: "systemd" is different from docker cgroup driver: "cgroupfs"
```

Comment:
`from docker` -> `from the Docker`

Reply:
This is the error output from the kubelet when the cgroup driver does not match.

Reply:
oh...